src/style.css

```diff
@@ -0,0 +1,4 @@
+@import url(./blocks/card.scss);
+@import url(./blocks/stars.scss);
+@import url(./normalize.css);
+@import url(./reset.css);
```

Comment:
Consider importing reset.css before normalize.css. Reset.css should remove all default styles, and then normalize.css can apply consistent styling across browsers 🌐